### ISSUE TITLE
Add pagination to GraphQL queries

### ISSUE THREAD

**Issue by yankeexe** [yankeexe/good-first-issues#11]
GitHub allows fetching `first: 100` or `last:100` items from the GraphQL API.
With that the CLI is also fetching data from the first 100 repos in an organization. 
Add pagination feature to load data from more pages.

Resource: [Pagination GraphQL](https://graphql.org/learn/pagination/)

**Comment by rexdivakar** [comment]
Assign it to me i shall work on it @yankeexe

**Comment by yankeexe** [comment]
Sure @rexdivakar sure 🙂

**Comment by yankeexe** [comment]
Hey @rexdivakar are you working on the issue? 🙂

**Comment by rexdivakar** [comment]
hey @yankeexe yes I'm will complete soon and raise PR

**Comment by yankeexe** [comment]
@rexdivakar  awesome!

**Comment by sravanth-chebrolu** [comment]
If no one's working and if it's up for grabs, I'm ready to take this up :)

**Comment by yankeexe** [comment]
@sravanth-chebrolu sure! You can work on it.

**Comment by sravanth-chebrolu** [comment]
So I've got the general idea, i.e. using the `endCursor` and `hasNextPage`, however I could use some advice/help on how I should go about implementing it. As in, how should the user request for the next set of query results via the command line?

Currently the CLI session ends as soon as the query finishes, but the the idea is to keep the session alive and request user input for additional pages(if the `hasNextPage` returns `True`), right?

**Comment by chebro** [comment]
@yankeexe

**Comment by yankeexe** [comment]
@chebro yeah you are right about the implementation detail. However, what kind of help do you need? Maybe we can look that over a PR? 🙂

**Comment by SaboorNisha** [comment]
can I work on this issue?

**Comment by SaboorNisha** [comment]
Thank you, @yankeexe! I will begin working on the PR shortly and Looking forward to contributing to the project! 😊
